src/features/dashboard/components/AppForm/index.tsx

```diff
@@ -69,13 +101,21 @@ const AppForm = ({
     </React.Fragment>
   );
 
+  const renderButtons = () => {
+    return (
+      <div className={styles.buttons}>
+        <Button role='submit' disabled={disable_btn}>
+          {button_text}
```

Comment:
Perhaps, we can remove the `button_text` prop and instead show the correct button text this way:

`is_update_mode ? 'Update Application' : 'Register Application'`